nbproject/packaging/macosx/productbuild/dist.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<installer-gui-script minSpecVersion="1">
+    <pkg-ref id="au.com.trgtd.tr">
+        <bundle-version/>
+    </pkg-ref>
+    <title>ThinkingRock Application</title>
+    <license file="tr-license.txt"/>
```

Comment:
In numerous project.properties files, we reference `nbproject/packaging/tr-license.txt`, which so far didn't exist in the repo (due to the entry in .gitignore). IMHO we should copy the LICENSE file in the root of the project to nbproject/packaging/tr-license.txt and then also use that copy from this file here. You could remove the empty file `nbproject/packaging/macosx/productbuild/tr-license.txt` in the PR.

Reply:
wouldn't the path to the license file here have to be adjusted to something like 

```suggestion
    <license file="../../../tr-license.txt"/>
```

? (I'm guessing here)...